src/components/Login/Login.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import {
+  View,
+  Text,
+} from 'react-native';
+import TextStyles from '../../helpers/TextStyles';
+import TextField from '../common/TextField';
+import ShadowStyles from '../../helpers/ShadowStyles';
+import Button from '../common/Button';
+import styles from './styles';
+import App from '../../App';
+
+class Login extends Component {
+  constructor(props) {
```

Comment:
Question: you prefer to always include props here even if not used? I prefer just putting props in the params if the constructor is going to use them. But I don't know if there's a convention about that.

Reply:
I've never thought of it, but since it's only necessary to pass them if you are going to use them inside the constructor, I think I prefer your approach